### PR TITLE
TPC-only CAF fcl for SBND data

### PR DESCRIPTION
**Pull request by wjdanswjddl**
## Description 

fcl to make TPC-only CAFs for SBND data. Changes are:

- objects updated to corresponding ones for data, `sptpc2d`, `pandoraCaloData` and `PandoraPidData`
- Dazzle/Razzle excluded from workflow, reason: module needs to be updated to remove sim product calls (ex. [this line](https://github.com/SBNSoftware/sbncode/blob/56dc342fed4f5947318a87fa4b887889558b8021/sbncode/PID/Dazzle_module.cc#L139))
- CRUMBS excluded from workflow, reason: requires other subsystems

## Checklist
- [x] Added at least 1 label from [available labels](https://github.com/SBNSoftware/sbndcode/issues/labels?sort=name-asc).
- [x] Assigned at least 1 reviewer under `Reviewers`,
- [x] Assigned all contributers including yourself under `Assignees`
- [x] Linked any relevant issues under `Developement`
- [x] Does this PR affect CAF data format? If so, please assign a CAF maintainer as additional reviewer.
- [x] Does this affect the standard workflow? 

### Relevant PR links (optional)
Does this PR require merging another PR in a different repository (such as sbnanobj/sbnobj etc.)?

### Link(s) to docdb describing changes (optional)
Is there a docdb describing the issue this solves or the feature added?

### CODE REVIEW

**Review by yangtj207 — APPROVED** [review]

File: sbndcode/JobConfigurations/standard/caf/cafmakerjob_sbnd_data.fcl [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+  ExptGeoHelperInterface:    @local::sbnd_geometry_helper
+  GeometryConfigurationWriter: {}
+  Geometry:                  @local::sbnd_geo
+  AuxDetGeometry:            @local::sbnd_auxdetgeo
+  LArPropertiesService:      @local::standard_properties
+  DetectorClocksService:     @local::sbnd_services.DetectorClocksService
+  DetectorPropertiesService: @local::sbnd_detproperties
+  ChannelStatusService:      @local::sbnd_channelstatus
+  ParticleInventoryService:  @local::sbnd_particleinventoryservice
+  BackTrackerService:        @local::sbnd_backtrackerservice
```
Comment (PetrilloAtWork) [inline sbndcode/JobConfigurations/standard/caf/cafmakerjob_sbnd_data.fcl:59]: These services are for tracking back to generated information. Are they needed here?
```suggestion
```

**Review by PetrilloAtWork — COMMENTED** [review]
Left some comments, nothing critical.

File: sbndcode/JobConfigurations/standard/caf/cafmakerjob_sbnd_data.fcl [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+   pandoraShowerSelectionVars:  @local::showerSelectionVars_sbn
+   pandoraShowerCosmicDist: @local::showerCosmicDist_sbn
+   pandoraShowerRazzle:         @local::razzle_sbnd
+
+   pandoraRazzled: @local::razzled_sbnd
+
+   systtools: @local::sbn_systtools
+}
+  filters:{
+    flashtrigfilter: @local::pmtflashtrigfilter_sbnd
```
[the remark this answers is not in the document]
Comment (wjdanswjddl) [reply sbndcode/JobConfigurations/standard/caf/cafmakerjob_sbnd_data.fcl:107]: yup we're using just TPC data for now

File: sbndcode/JobConfigurations/standard/caf/cafmakerjob_sbnd_data.fcl [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+process_name: CAFmaker
+
+services:
+{
+  # Load the service that manages root files for histograms.
+#  scheduler:    {wantTracer : false wantSummary : false}
+  TimeTracker: {}
+  RandomNumberGenerator: {} #ART native random number generator
+  TFileService: { fileName: "fullchain_production_hist_prod5.root" closeFileFast: false }
```
[the remark this answers is not in the document]
Comment (wjdanswjddl) [reply sbndcode/JobConfigurations/standard/caf/cafmakerjob_sbnd_data.fcl:43]: It's just what the MC caf fcl does -- don't know the reasoning behind it. If @JosiePaton agrees I can change it to `%ifb`

**Review by sungbinoh — APPROVED** [review]
Thanks for updating labels for pandoraCalo and Pid.
It looks good for me!

**Comment by henrylay97** [comment]
@wjdanswjddl Can the problem with the Razzle/Dazzle/Razzled modules be resolved by setting the `MakeTree` fcl parameter to false for each one? This is setup to make the trees for training, hence the use of truth information.

File: sbndcode/JobConfigurations/standard/caf/cafmakerjob_sbnd_data.fcl (hunk 3 above)
[the remark this answers is not in the document]
Comment (wjdanswjddl) [reply sbndcode/JobConfigurations/standard/caf/cafmakerjob_sbnd_data.fcl]: I've included %ifb. I think it shouldn't change anything but @JosiePaton can confirm.

**Comment by wjdanswjddl** [comment]
@bear-is-asleep PR is ready to be merged

**Comment by wjdanswjddl** [comment]
> @wjdanswjddl Can the problem with the Razzle/Dazzle/Razzled modules be resolved by setting the `MakeTree` fcl parameter to false for each one? This is setup to make the trees for training, hence the use of truth information.

Looks like this is already set to false, and the module asks for sim info anyways. I propose we close this PR for now since Razzle require light information (I think) which isn't included anyway and Dazzle is not being actively used -- and we can revisit this at the next iteration for data processing.

**Comment by henrylay97** [comment]
@wjdanswjddl Thanks for looking! Yeah can definitely wait :)

**Comment by bear-is-asleep** [comment]
trigger build LArSoft/lar*@LARSOFT_SUITE_v09_91_02 https://github.com/SBNSoftware/sbncode/commit/56dc342f https://github.com/SBNSoftware/sbnobj/commit/a4b93770 https://github.com/SBNSoftware/sbndcode/commit/e303cbe2 SBNSoftware/sbnanaobj@bug/coackley_ciCafv09_91_02

**Comment by bear-is-asleep** [comment]
trigger build LArSoft/lar*@LARSOFT_SUITE_v09_91_02 SBNSoftware/sbnanaobj@bug/coackley_ciCafv09_91_02

**Comment by bear-is-asleep** [comment]
trigger build LArSoft/lar*@LARSOFT_SUITE_v09_91_02 SBNSoftware/sbnanaobj@bug/coackley_ciCafv09_91_02 SBNSoftware/sbncode_v09_91_02_02

**Comment by bear-is-asleep** [comment]
trigger build LArSoft/lar*@LARSOFT_SUITE_v09_92_00 SBNSoftware/sbnanaobj@bug/coackley_ciCafv09_91_02

**Comment by bear-is-asleep** [comment]
trigger build LArSoft/lar*@LARSOFT_SUITE_v09_93_01 SBNSoftware/sbnanaobj@bug/coackley_ciCafv09_91_02

**Comment by bear-is-asleep** [comment]
Ready for merge

**Pull request merged**